js-miniapp-sdk/test/miniapp.spec.ts

```diff
@@ -42,3 +45,14 @@ describe('requestPermission', () => {
     );
   });
 });
+
+describe('showInterstitialAd', () => {
+  it('should retrieve show interstitial ad resolves specific InterstitialAdResponse response result from the Mini App Bridge', () => {
```

Comment:
```suggestion
describe('showInterstitialAd', () => {
  it('should retrieve InterstitialAdResponse type of result from the Mini App Bridge', () => {
```

Reply:
There should be further cases to unit test the success and failure case. What do you think?